src/components/HappThumbnail/HappThumbnail.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import './HappThumbnail.module.css'
+
+const HappThumbnail = ({ title, url, className }) => (
```

Comment:
I would have held off on making this a component yet.

Reply:
Yeah, this didn't help much, but it just looked obvious to me given the designs we have right now.
Now that it's done, I don't see a reason to revert, unless you feel strongly about it. ;)

Reply:
don't feel strongly, I think it's fine to leave it like this. Just noting a slight difference in approach.